scripts/generate-global-changelog.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * SPDX-FileCopyrightText: © 2021 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const {execSync} = require('child_process');
+const fs = require('fs');
+const path = require('path');
+
+const IGNORE_LIST = [
+	'packages/demos/CHANGELOG.md',
+	'clayui.com/CHANGELOG.md',
+	'CHANGELOG.md',
+];
+
+const dateObj = new Date();
+const month = dateObj.getMonth() + 1;
+const day = dateObj.getDate();
+const year = dateObj.getFullYear();
+
+const dateString = `${year}-${month}-${day}`;
+
+const globalChangeLog = path.join(__dirname, '../CHANGELOG.md');
+
+function main() {
+	const diffOutput = execSync(
+		'git diff HEAD^ --name-only -- "*CHANGELOG.md"'
```

Comment:
You can get rid of the `IGNORE_LIST` and make this less imprecise if you switch to something like:

```
git diff HEAD^ --name-only -- 'packages/clay-*/CHANGELOG.md'
```